src/burnchains/burnchain.rs

```diff
@@ -321,10 +340,20 @@ impl Burnchain {
             consensus_hash_lifetime: params.consensus_hash_lifetime,
             stable_confirmations: params.stable_confirmations,
             first_block_height: params.first_block_height,
-            first_block_hash: params.first_block_hash.clone()
+            first_block_hash: params.first_block_hash.clone(),
+            reward_cycle_period: 1000,
```

Comment:
Is there also a separate registration period?  Or, is the registration period pipelined with the last PoX reward cycle?

Reply:
The actual calculation of the anchor block isn't done in this PR, so those constants aren't added to the `Burnchain` here -- in the PR I'll open for anchor block detection, there's a `PoxConstants` struct used in the `Burnchain` struct which holds the reward cycle length, the prepare phase length, and the anchor block threshold.

Reply:
Sounds good.  I'll be sure to add a stub in my PR for lockups which will eventually just read this value.